browser/src/control/Signing.js

```diff
@@ -16,43 +16,56 @@
 
 L.Map.include({
 	onChangeSignStatus: function(signstatus) {
-		var statusText = '';
-		var statusIcon = '';
+		// Have a non-empty status text by default, so a signatures -> no signatures
+		// transition updates the status bar.
+		var statusText = 'The document is not signed.';
```

Comment:
Does this need the _() markup so it gets translated?

Reply:
Yes, let me fix.